reloc.ml

```diff
@@ -898,6 +914,28 @@ let build_dll link_exe output_file files exts extra_args =
     )
     libobjects;
 
+    begin match !imports_file with
+      | None -> ()
+      | Some file ->
+        let oc = open_out file in
+        StrSet.iter (Printf.fprintf oc "%s\n") !imported_all;
+        close_out oc
+    end;
+
+    begin match !exports_files with
+      | [] -> ()
+      | files ->
+        let ht = Hashtbl.create 7 in
+        let exists s = try Hashtbl.find ht s with Not_found -> false in
+        List.iter
+          (fun file ->
+              List.iter
+                (fun name -> if not (exists name) then Hashtbl.add ht name true)
```

Comment:
This could just be: `(fun name -> Hashtbl.replace ht name ())`

Reply:
Indeed; sometimes old code looks really strange :) Will fix that.